mobile/test/cc/unit/envoy_config_test.cc

```diff
@@ -65,8 +65,8 @@ bool socketAddressesEqual(
 
 TEST(TestConfig, ConfigIsApplied) {
   EngineBuilder engine_builder;
-  engine_builder.setHttp3ConnectionOptions("5RTO")
-      .setHttp3ClientConnectionOptions("MPQC")
+  engine_builder.addQuicConnectionOption("10AF")
+      .addQuicConnectionOption("MPQC")
```

Comment:
It would be nice to test the client connection option test as well

Reply:
Sorry it auto-merged before I could add the test, so I added it in a new PR: https://github.com/envoyproxy/envoy/pull/42810